internal/patroni/reconcile.go

```diff
@@ -108,6 +108,13 @@ func InstancePod(ctx context.Context,
 	// "kubernetes.labels" settings.
 	outInstancePod.Labels[naming.LabelPatroni] = naming.PatroniScope(inCluster)
 
+	// Add annotations to get metrics from the Pod
+	if outInstancePod.Annotations == nil {
+		outInstancePod.Annotations = make(map[string]string)
+	}
+	outInstancePod.Annotations["prometheus.io/scrape"] = "true"
+	outInstancePod.Annotations["prometheus.io/port"] = "8008"
```

Comment:
Does this need a path annotation for `/metrics`?

Reply:
nope, its the default in fks